boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/liberty/LibertyPackageMojo.java

```diff
@@ -78,51 +84,71 @@ public void execute() throws MojoExecutionException {
-                        .getSpringBootMavenPluginClassifier(project, getLog());
-
-                // Check if we need to attach based on the classifier configuration
-                if (MavenProjectUtil.isNotNullOrEmpty(springBootClassifier)) {
-                    attach = false;
-                } else {
-                    attach = true;
-                }
-
-                File springBootUberJar = net.wasdev.wlp.maven.plugins.utils.SpringBootUtil.getSpringBootUberJAR(project,
-                        getLog());
-                validateSpringBootUberJAR(springBootUberJar);
-                copySpringBootUberJar(springBootUberJar, attach); // Only copy back if we need to overwrite the project
-                                                                  // artifact
-                generateServerXML();
-                generateBootstrapProps();
-                installMissingFeatures();
-                installApp("spring-boot-project");
-
-                if (springBootUberJar != null) {
-                    // Create the Liberty Uber JAR from the Spring Boot Uber JAR in place
-                    createUberJar(springBootUberJar.getAbsolutePath(), attach);
-                } else {
-                    // The Spring Boot Uber JAR was already replaced with the Liberty Uber JAR (this
-                    // is a re-execution in the non-classifier scenario)
-                    createUberJar(null, attach);
-                }
+        if (MavenProjectUtil.isNotNullOrEmpty(springBootVersion)) { // Dealing with a spring boot app
+            springBootClassifier = net.wasdev.wlp.maven.plugins.utils.SpringBootUtil
+                    .getSpringBootMavenPluginClassifier(project, getLog());
 
-                if (!MavenProjectUtil.isNotNullOrEmpty(springBootClassifier)) {
-                    // If necessary, add the manifest to prevent Spring Boot from repackaging again
-                    addSpringBootVersionToManifest(springBootVersion);
-                }
-            } else { // Dealing with an EE based app
+            // Check if we need to attach based on the classifier configuration
+            if (MavenProjectUtil.isNotNullOrEmpty(springBootClassifier)) {
                 attach = false;
-                boosterFeatures = getBoosterConfigsFromDependencies(project);
-                generateServerXMLJ2EE(boosterFeatures);
-                installMissingFeatures();
-                installApp("project");
+            } else {
+                attach = true;
+            }
 
+            File springBootUberJar = net.wasdev.wlp.maven.plugins.utils.SpringBootUtil.getSpringBootUberJAR(project,
+                    getLog());
+            validateSpringBootUberJAR(springBootUberJar);
+            copySpringBootUberJar(springBootUberJar, attach); // Only copy back if we need to overwrite the project
+                                                              // artifact
+
+            generateServerConfigSpringBoot();
```

Comment:
The diff formatting is a bit weird in this method so it's hard to see what exactly was changed. So to clarify, the only thing that was changed here was to remove the try/catch and replace the `generateServerXML();` and `generateBootstrapProps();` methods with `generateServerConfigSpringBoot();`, is that correct?

Reply:
He covered this with his comment on the PR.

Reply:
Correct - for some reason, git butchered this (I think because the MojoException moved).